service/history/replication/task_ack_manager.go

```diff
@@ -196,6 +200,7 @@ TaskInfoLoop:
 		}
 		readLevel = taskInfo.GetTaskID()
 		if replicationTask != nil {
+			taskSize += replicationTask.GetSizeInByte()
```

Comment:
Will we have the same size limit issue when returning the aggregated response from history to frontend?

Reply:
The aggregated response operation happens in history. If the size exceeds, the response will be dropped